docs/src/config.ts

```diff
@@ -0,0 +1,5 @@
+const config = {
+  getStatsUri: 'https://stardust.azurewebsites.net/api/GetPerfStats',
```

Comment:
We already have a `config.ts` in root, do we really need a separate config?

Reply:
Not sure with this one. I know we have a global config, but this one is docs-specific. Should it really go to the global one?

Reply:
Don't have a strong opinion there, but I don't reasonable need for it... You can go with your decision 👍